xmtp_mls/benches/group_limit.rs

```diff
@@ -172,7 +177,7 @@ fn add_to_100_member_group_by_inbox_id(c: &mut Criterion) {
                             .add_members_by_inbox_id(
                                 // it is OK to take from the back for now because we aren't getting
                                 // near MAX_IDENTITIES
-                                inbox_ids.iter().rev().take(100).cloned().collect(),
+                                &inbox_ids,
```

Comment:
Do we know why we were originally doing this 100 at a time? If not, we should probably leave it as-is

Reply:
We're still taking 100 at a time. https://github.com/xmtp/libxmtp/pull/1152/files/829289369ec1172c170929014df33eade5e8ab2b#diff-cd11ded27b36e44d16af4f4f8426b315cba5e17487db308d9d6a92483ff9fb9bR156

Reply:
Or are you saying we would rather leave it in the loop so that the benches aren't distorted?

Reply:
Nevermind. This looks fine

Reply:
This is just setup for the benchmark, its taking 100 from the back of a very large generated list of identities either on local or dev. The benchmark itself is progressing by the predetermined steps from the front of the list. Moving the iteration out of here should be fine, since its not the part being benchmarked